browser/app/js/browser/BrowserDropdown.js

```diff
@@ -118,20 +108,9 @@ export class BrowserDropdown extends React.Component {
                 />
               )}
             </li>
-            <li>
-              <a href="" onClick={this.showChangePassword.bind(this)}>
-                Change Password <i className="fas fa-cog" />
-              </a>
-              {this.state.showChangePasswordModal && (
-                <ChangePasswordModal
-                  serverInfo={serverInfo}
-                  hideChangePassword={this.hideChangePassword.bind(this)}
-                />
-              )}
-            </li>
             <li>
               <a href="" id="logout" onClick={this.logout}>
-                Sign Out <i className="fas fa-sign-out-alt" />
+                Logout <i className="fas fa-sign-out-alt" />
```

Comment:
If this is Logout, then we should have LogIn in the login screen with `alt="Login"`

Reply:
Hi @harshavardhana, from what I have understood, you want me to apply the `alt` attribute to the Login button on the main page. However, it seems like the [`alt` attribute can only be used in `img`, `area` and `input` tags](https://www.w3.org/QA/Tips/altAttribute), and the Login button only contains the `button` and `i` tags where the attribute isn't available. Where exactly should I be applying this attribute? Thanks in advance!